yb-voyager/cmd/importData.go

```diff
@@ -160,11 +160,17 @@ func startExportDataFromTargetIfRequired() {
 	importTableList := getImportTableList(tableListExportedFromSource)
 
 	lockFile.Unlock() // unlock export dir from import data cmd before switching current process to ff/fb sync cmd
+
+	if tconf.SSLMode == "prefer" || tconf.SSLMode == "allow" {
+		utils.PrintAndLog(color.RedString("Warning: SSL mode '%s' is not supported for 'export data from target' yet. Downgrading it to 'disable'.\nIf you dont want these settings you can restart the 'export data from target' with the --target-ssl-mode flag.", source.SSLMode))
```

Comment:
If you dont want these settings you can restart the 'export data from target' with a different value for --target-ssl-mode flag.